### ISSUE TITLE
`tpoint_trajectory` error for TPointSequenceSets with stationary sequence

### ISSUE THREAD

**Issue by Diviloper** [MobilityDB/MobilityDB#539]
**Describe the bug**
When calling `tpoint_trajectory` with a TPoint SequenceSet with one stationary sequence (same position throught all the sequence), it fails with : `Unknown geometry type: 0 - Unknown`.  
This error only happens if there is more than one sequence and one of them has the same position for every instant:
- Stationary sequence `{[Point(1 1)@2019-09-01, Point(2 2)@2019-09-02],[Point(1 1)@2019-09-03, Point(1 1)@2019-09-05]}`: 
    `Unknown geometry type: 0 - Unknown`
- Instantaneous sequence `{[Point(1 1)@2019-09-01, Point(2 2)@2019-09-02],[Point(1 1)@2019-09-03]}`: 
    `Unknown geometry type: 0 - Unknown`
- Non-stationary sequences `{[Point(1 1)@2019-09-01, Point(2 2)@2019-09-02],[Point(1 1)@2019-09-03, Point(2 2)@2019-09-05]}`: 
    `MULTILINESTRING((1 1,2 2),(1 1,2 2))`
- Partially stationary sequence `{[Point(1 1)@2019-09-01, Point(1 2)@2019-09-02],[Point(1 1)@2019-09-03, Point(1 1)@2019-09-04, Point(2 2)@2019-09-05]}`: 
    `MULTILINESTRING((1 1,2 2),(1 1,2 2))`
- Single stationary sequence `{[Point(1 1)@2019-09-03, Point(1 1)@2019-09-05]}`: 
    `POINT(1 1)`

The issue seems to be happening when the resulting geometry would combine Linestrings and points, but I'm not really sure.

**To Reproduce**
Run the following C code:
```c
#include <meos.h>

int main() {
    meos_initialize(NULL, NULL);

    Temporal *tp = tgeompoint_in("{[Point(1 1)@2019-09-01, Point(2 2)@2019-09-02],[Point(1 1)@2019-09-03, Point(1 1)@2019-09-05]}");
    printf("Temporal: %s\n", tpoint_as_text(tp, 3));
    GSERIALIZED *gs = tpoint_trajectory(tp);
    printf("%s\n", geo_as_ewkt(gs, 2));
    meos_finalize();

    return 0;
}
```

**Expected behavior**
A valid geometry should be returned.

**Specifications (please complete the following information):**
MEOS library, master branch (6b60876817b53bc33967f7df50ab8e1f482b0133)

**Comment by mschoema** [comment]
Thanks for notifying this, I'm taking a look at it.

**Comment by mschoema** [comment]
I believe the issue has been fixed in #540, but you're welcome to reopen the issue if something is still wrong.

**Comment by Diviloper** [comment]
Great! Thanks! 
I'll run the tests again and come back here if anything fails.
